src/serialization/SerializationSymbols.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @ignore *//** */
+
+import * as Long from 'long';
+
+/** @internal */
+export class SerializationSymbols {
+    static readonly BYTE_SYMBOL = Symbol();
+    static readonly NULL_SYMBOL = Symbol();
+    static readonly SHORT_SYMBOL = Symbol();
+    static readonly INTEGER_SYMBOL = Symbol();
+    static readonly LONG_SYMBOL = Symbol();
+    static readonly FLOAT_SYMBOL = Symbol();
+    static readonly CHAR_SYMBOL = Symbol();
+    static readonly JAVACLASS_SYMBOL = Symbol();
+    static readonly ARRAYLIST_SYMBOL = Symbol();
+    static readonly LINKLIST_SYMBOL = Symbol();
+    static readonly JAVA_ARRAY_SYMBOL = Symbol();
+    static readonly COMPACT_SYMBOL = Symbol();
+    static readonly IDENTIFIED_SYMBOL = Symbol();
+    static readonly PORTABLE_SYMBOL = Symbol();
+    static readonly JSON_SYMBOL = Symbol();
+    static readonly GLOBAL_SYMBOL = Symbol();
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-types 
+export function getTypes(clazz: Function | Symbol): Function | Symbol {
```

Comment:
If you read all my comments and apply them, you will see that this method won't be required because all it does will be `return clazz === Buffer ? return SerializationSymbols.BYTE_SYMBOL : clazz`

Reply:
I remove this method and check inside the "findSerializerByType" method.